src/main/webapp/assets/modules/common/securityService.js

```diff
@@ -114,73 +99,57 @@ techlooper.factory("securityService", function (apiService, $rootScope, $q, util
       }
     },
 
-    removeProtectedLastFoot: function () {
-      var path = localStorageService.get("protectedPage");
-      if (instance.isProtectedPage(path)) {
-        localStorageService.remove("protectedPage");
-      }
-    },
-
-    isProtectedPage: function (path) {
-      //var path = localStorageService.get("protectedPage");
-      switch (utils.getView(path)) {
-        case jsonValue.views.freelancerPostProject:
-          return "/freelancer/post-project";
+    initialize: function () {
+      $rootScope.$on("$locationChangeSuccess", function (event, next, current) {
```

Comment:
'current' is defined but never used.<br>'next' is defined but never used.<br>'event' is defined but never used.